index.html

```diff
@@ -10,7 +10,10 @@
     <link rel="stylesheet" href="css/style.css">
 </head>
     <body>
-
+      <div id = "Nav"></div>
```

Comment:
make sure your capitalization is consistent.

Also sorry to hear about the loss of your `igation` keys 😢